test/test-lex.cpp

```diff
@@ -649,10 +649,29 @@ TEST_F(test_lex, lex_strings) {
                                 offsets_matcher(input, 0, 14))));
       });
 
-  // TODO(strager): Report invalid hex escape sequences. For example:
```

Comment:
Also include a positive test case (around line 550).

Reply:
Added two strings with hex escapes and no errors, not sure if you had something else in mind.